HousingRegisterApi/V1/Gateways/DynamoDbGateway.cs

```diff
@@ -135,6 +138,8 @@ public Application UpdateApplication(Guid id, UpdateApplicationRequest request)
             if (request.Assessment != null)
                 entity.Assessment = request.Assessment;
 
+            entity.CalculatedBedroomNeed = _bedroomCalculatorService.Calculate(entity.ToDomain());
```

Comment:
This should be added to the create application method as well? since the data provided might require a calculation.

Reply:
done